Apps/Common/src/AccessManagement/Authentication/Models/IAuthModel.cs

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //-------------------------------------------------------------------------
-namespace HealthGateway.Common.Authentication.Models
+namespace HealthGateway.Common.AccessManagement.Authentication.Models
```

Comment:
This model does not look like it needs an interface. It can be implemented directly